src/main/java/lotto/domain/result/LottoResult.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain.result;
+
+import java.util.Map;
+
+import lotto.domain.money.LottoMoney;
+
+public class LottoResult {
+
+    private static final int HUNDRED_PERCENTAGE = 100;
+
+    private Map<Rank, Integer> rankMap;
+
+    public LottoResult(Map<Rank, Integer> rankMap) {
+        this.rankMap = rankMap;
+    }
+
+    public int count(Rank rank) {
+        return rankMap.get(rank);
+    }
+
+    public double getProfit(LottoMoney lottoMoney) {
+        double profit = getTotalPrize() - lottoMoney.getMoney();
+        double profitRatio = profit / lottoMoney.getMoney();
+        return profitRatio * HUNDRED_PERCENTAGE;
+    }
+
+    private double getTotalPrize() {
+        long totalPrize = 0;
```

Comment:
Stream을 이용하면 아래처럼 할 수 있어요. (참고하시라고 말씀드려요 ㅎㅎ)
```suggestion
return rankMap.keySet().stream()
                .mapToLong(rank -> rank.getWinningMoneyByCount(rankMap.get(rank)))
                .sum();
```

Reply:
메서드를 만들어서 할 생각을 못했었습니다.
말씀해주신대로 수정했습니다.